src/libraries/System.Text.Json/tests/Serialization/ContinuationTests.NullToken.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    partial class ContinuationTests
+    {
+        // From https://github.com/dotnet/runtime/issues/42070
+        [Theory]
+        [MemberData(nameof(ContinuationAtNullTokenTestData))]
+        public static async Task ContinuationAtNullToken(string payload)
+        {
+            using (Stream stream = new MemoryStream(Encoding.UTF8.GetBytes(payload)))
+            {
+                CustomerCollectionResponse response = await JsonSerializer.DeserializeAsync<CustomerCollectionResponse>(stream, new JsonSerializerOptions { IgnoreNullValues = true });
+                Assert.Equal(50, response.Customers.Count);
+            }
+        }
+
+        public static IEnumerable<object[]> ContinuationAtNullTokenTestData
+            => new[]
+            {
+                new[] { SR.CustomerSearchApi108KB },
```

Comment:
These large resources take up a lot of disk space; is there a way to produce at run-time?

@layomia perhaps we need to take this offline since there are other test resources that are quite large (like `Json400K`)

Reply:
See #42428